.github/workflows/security-scan-weekly.yaml

```diff
@@ -82,3 +117,21 @@ jobs:
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: 'jimm-container-trivy-results.sarif'
+
+      - name: Fetch KEV list from CISA
+        shell: bash
+        run: |
+          curl -s -o ./kev.json https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json
+
+      - name: Compare Trivy container scan results with KEV list
+        run: ./scripts/compare_kev_vulnerabilities.sh 2>&1 | tee kev-trivy-container-scan.txt
+        shell: bash
+        env:
+          VULN_REPORT_FILE: jimm-container-trivy-results.sarif
+          KNOWN_CVES_FILE: kev.json
+
+      - name: Upload KEV Trivy Container scan results as artifact
```

Comment:
It may be ok to fail to workflow. KEVs should not appear often and when they do, is likely ok to block further merges until it's fixed. Otherwise further merges could potentially (even if very unlikely) further exacerbate the KEV.

Reply:
Yeah in a follow-up I changed it to exit 1 if a kev is detected.